extensions/adobe/experience/customerJourneyManagement/email-channel-context.schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/customerJourneyManagement/email-channel-context",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe CJM ExperienceEvent - MessageProfile Email Channel Details",
+  "description": "Email channel fields for the MessageProfile associated with the Adobe CJM Experience Event.",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+
+  "definitions": {
+    "emailChannelContext": {
+      "properties": {
+        "https://ns.adobe.com/experience/customerJourneyManagement/emailChannelContext/address": {
+          "title": "Address",
+          "type": "string",
+          "format": "email",
+          "description": "The technical address, for example, 'name@domain.com' as commonly defined in RFC2822 and subsequent standards."
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/emailChannelContext/operationalContext": {
+          "title": "Operational Context",
+          "type": "object",
+          "properties": {
+            "https://ns.adobe.com/experience/customerJourneyManagement/emailChannelContext/outboundIP": {
+              "title": "Outbound IP Address",
+              "type": "string",
+              "description": "Outbound IP address of the customer used to deliver the message.",
+              "oneOf": [
```

Comment:
oneOf not supported in AEP. This will throw an error when you try to use the schema there.